test/base/test_svd.jl

```diff
@@ -186,6 +186,51 @@ include(joinpath(@__DIR__, "utils", "util.jl"))
     @test Base.eltype(V) === eltype
   end
 
+  @testset "svd arrow directions" begin
+
+    #Test Spins
+    n = 10
+    χ = 5
+    s = siteinds("S=1/2", n; conserve_qns=true)
+    state = [isodd(i) ? "Up" : "Dn" for i in 1:n]
+    ψ = randomMPS(s, state; linkdims=χ)
```

Comment:
Can we test this without using `randomMPS` (i.e. without using the MPS type)? We're trying to avoid that in "basic" tests since we plan to move the MPS/MPO code out of the package once `ITensorNetworks` is more developed.